### PR TITLE
Add jsonapi-fractal

### PR DESCRIPTION
**Pull request by andersondanilo**

### CODE REVIEW

File: implementations/index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ assembled to vet them.
 * [hapi-json-api](https://github.com/wraithgar/hapi-json-api) Plugin for the hapi framework; enforces Accept/Content-type rules and rewrites Boom errors to be spec compliant.
 * [jsonapi-datastore](https://github.com/beauby/jsonapi-datastore) is a lightweight standalone library for reading, serializing, and synchronizing relational JSON:API data.
 * [json-api-store](https://github.com/haydn/json-api-store) A lightweight JavaScript library for using JSON:API in the browser.
+* [jsonapi-fractal](https://github.com/andersondanilo/jsonapi-fractal) JSON:API Serializer inspired by Fractal (PHP)
```
Comment (dgeb) [inline implementations/index.md]: Could you please move this to the bottom of the list? We try to keep these sections ordered chronologically.

Comment (andersondanilo) [reply implementations/index.md]: Ok, i changed

Comment (dgeb) [reply implementations/index.md]: Thanks for updating. There's a conflict that needs to be resolved. Could you please rebase on `gh-pages` please (or open a new PR, whichever is easier for you)?

Comment (andersondanilo) [reply implementations/index.md]: Ok

**Comment by dgeb** [comment]
Thanks @andersondanilo!

**Pull request merged**